src/components/UserProfile/QuickSetupModal/QuickSetupModal.jsx

```diff
@@ -20,9 +20,9 @@ function QuickSetupModal({
   setUserProfile,
   handleSubmit,
   setSaved,
+  canViewTitle
```

Comment:
```suggestion
```
Instead of importing this as a prop, `connect()` this component, add `hasPermission()` to `mapDispatchToProps`, and memoise the `props.hasPermission()` call.

This should help if you have any questions:
https://docs.google.com/document/d/1rzCvlu5FzIXoUpWIM07Z5g80UgqN2DikT2Csz1FdtV8/edit